contracts/PricerInterface.sol

```diff
@@ -1,3 +1,4 @@
+/* solhint-disable-next-line compiler-fixed */
```

Comment:
There is no need for `PricerINterface.sol`, please remove

Reply:
as we concluded we will keep it.